tools/ClassGen/Backends/OpenCL/OpenCLSpecificNodesVerification.h

```diff
@@ -30,4 +30,65 @@ bool OCLConvolutionNode::verify() const {
 bool OCLAvgPoolNode::verify() const { return true; }
 
 bool OCLMaxPoolNode::verify() const { return true; }
+
+bool OCLBatchedReduceAddNode::verify() const {
```

Comment:
Can we use `expectCompareTrue` and other helpers to make this verifier not only return false in case of failures, but produce some user-friendly error messages?